src/taskgraph/run-task/run-task

```diff
@@ -1090,7 +1090,7 @@ def main(args):
     # Sort repositories so that parent checkout paths come before children
     repositories.sort(key=lambda repo: Path(repo["checkout"] or "/").parts)
 
-    uid = gid = gids = None
+    uid = gid = gids = user = group = None
```

Comment:
From a quick look it seems we could do without user and group and just pass uids around (the only thing that uses pw_name/gr_name is a logging message in chown_recursive, which could just as well use numeric IDs.

Reply:
I really only added these here so my IDE would stop complaining it wasn't set on line 1131